spec/linter-stylelint-spec.js

```diff
@@ -42,15 +47,15 @@ describe('The stylelint provider for Linter', () => {
     );
   });
 
-  it('bundles and works with stylelint-config-standard', () => {
```

Comment:
Here is are missing base stylelint-config-standard. Should be stylelint-config-standard and less stylelint-config-standard assertions.